frontend/app/src/components/helper/Card.vue

```diff
@@ -46,39 +44,26 @@ const otherHeights = computed(() => {
     v-bind="rootAttrs"
     :flat="flat"
     :class="{
-      [css['no-radius-bottom']]: noRadiusBottom,
-      [css['full-height']]: fullHeight
+      ['h-full']: fullHeight
     }"
+    :outlined="outlined"
     v-on="rootListeners"
   >
-    <VCardTitle v-if="slots.title" :class="{ 'pt-6': slots.icon }">
-      <slot v-if="slots.icon" name="icon" />
```

Comment:
We don't use the slot anywhere? app or components?

Reply:
I replace it to other component. So it's only used in LP (I put it in LpPoolIcon), because I feel the style is hacky, and all usage can be moved there.